modules/apps/segments/segments-context-vocabulary/src/main/java/com/liferay/segments/context/vocabulary/internal/field/customizer/SegmentsContextVocabularySegmentsFieldCustomizer.java

```diff
@@ -125,10 +125,11 @@ public String getLabel(String fieldName, Locale locale) {
 	@Activate
 	@Modified
 	protected void activate(Map<String, Object> properties) {
-		SegmentsContextVocabularyConfiguration
+		SegmentsContextVocabularyCompanyConfiguration
 			segmentsContextVocabularyConfiguration =
 				ConfigurableUtil.createConfigurable(
-					SegmentsContextVocabularyConfiguration.class, properties);
```

Comment:
You should read both, company and system configuration.

Reply:
Probably you will need a Configuration Provider, the same way we have in PageAudit